sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -465,7 +498,7 @@ public partial class ScriptCmdletData : Azure.ResourceManager.Models.ResourceDat
     {
         public ScriptCmdletData() { }
         public string Description { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Avs.Models.ScriptParameter> Parameters { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Avs.Models.ScriptParameterProperties> Parameters { get { throw null; } }
```

Comment:
can we rename `ScriptParameterProperties` to simplify this to just `ScriptParameter`?

Reply:
Actually, the original name is just `ScriptParameter` and I have reverted my renaming.